modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/QueryChecker.java

```diff
@@ -264,6 +267,16 @@ public QueryChecker returns(Object... res) {
         return CoreMatchers.containsString("IgniteUnionAll(all=[" + all + "])");
     }
 
+    /** */
+    public QueryChecker columnNames(String... columns) {
```

Comment:
I would prefer to override expected column names each time this method invoked. I can hardly imagine using it like this:
```
assertQuery("select c1, c2, c3, c4 from my_table")
    .columnNames("c1", "c2")
    .columnNames("c3", "c4")
    .check()
```

Reply:
I can't imagine a case when it required override %)